test/uk/gov/hmrc/plasticpackagingtax/registration/views/organisation/SoleTraderVerificationFailureViewSpec.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.views.organisation
+
+import base.unit.UnitViewSpec
+import org.scalatest.matchers.must.Matchers
+import play.twirl.api.Html
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.organisation.sole_trader_verification_failure_page
+import uk.gov.hmrc.plasticpackagingtax.registration.views.tags.ViewTest
+
+@ViewTest
+class SoleTraderVerificationFailureViewSpec extends UnitViewSpec with Matchers {
+
+  private val page: sole_trader_verification_failure_page =
+    instanceOf[sole_trader_verification_failure_page]
+
+  private def createView(): Html = page()(journeyRequest, messages)
+
+  "Sole Trader Verification Failure Page" should {
+
+    val view: Html = createView()
+
+    // Minimal asserts to verify the page is specific to sole traders and refers to self assessment
```

Comment:
Potential controversy here; just enough asserting to confirm we're talking about sole traders and self assessment; don't need to assert every message.